controllers/builders/roles.go

```diff
@@ -0,0 +1,59 @@
+package builders
+
+import (
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func CreateClusterRole(crdName string, crdNamespace string) *rbacv1.ClusterRole {
```

Comment:
This is actually wrong.  Binoculars and executor do not have similar roles at all.  Should revert this.